whats_fresh/whats_fresh_api/templates/story.html

```diff
@@ -123,7 +138,122 @@ <h4 id='error_tag' class='light warning'></h4>
 {% endblock content %}
 
 {% block footer %}
+<script src="//ajax.googleapis.com/ajax/libs/jquery/1.8.3/jquery.min.js"></script>
+<script src="//ajax.googleapis.com/ajax/libs/jqueryui/1.8.23/jquery-ui.min.js"></script>
+<link href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.8.23/themes/smoothness/jquery-ui.css" rel="stylesheet" type="text/css" media="all" />
 <script>
+
+        if ( window.FormData === undefined )
```

Comment:
@tschuy As you mentioned, I hope this would take care of the browsers that do not support FormData.